### PR TITLE
feat(tasks): add lint tasks

### PR DESCRIPTION
**Pull request by zewa666**
adds lint tasks for js and ts using respectively eslint or tslint

### CODE REVIEW

**Comment by zewa666** [comment]
this one just adds the new lint tasks but its not yet wired up to be copied during scaffolding of a new app.

Related issues https://github.com/aurelia/cli/issues/453 and https://github.com/aurelia/cli/issues/452

File: lib/resources/tasks/lint.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import * as gulp from 'gulp';
+import tslint from 'gulp-tslint';
+import * as project from '../aurelia.json';
+
+export default function lint() {
+  return gulp.src([project.transpiler.source])
```
Comment (JeroenVinke) [inline lib/resources/tasks/lint.ts]: source can be an array, will that cause issues?

**Comment by JeroenVinke** [comment]
Great! We can add the tasks [here](https://github.com/aurelia/cli/blob/master/lib/commands/new/buildsystems/cli/transpilers/babel.js#L11-L28) and also any dependencies that the task has. And the equivalent for [typescript](https://github.com/aurelia/cli/blob/master/lib/commands/new/buildsystems/cli/transpilers/typescript.js)

File: lib/resources/tasks/lint.ts (hunk 1 above)
Comment (zewa666) [inline lib/resources/tasks/lint.ts]: Nope this works ok. The reason is that I personally prefer to lint my unit tests as well so I'd typically add them here too. Not sure though if that counts for the masses. What would you say?

**Comment by zewa666** [comment]
Great, I'll make an update to include that.

**Comment by gheoan** [comment]
Running `aurelia help` does not show `lint` as a valid command. How can that be fixed?

**Comment by JeroenVinke** [comment]
@gheoan the lint.json is in the cli but doesn't get copied to the project directory. So we need a [similar line](https://github.com/zewa666/cli/blob/e16c2166dc962640b8041ca230fa00f5f61bd84a/lib/commands/new/buildsystems/cli/index.js#L120) in [here](https://github.com/zewa666/cli/blob/e16c2166dc962640b8041ca230fa00f5f61bd84a/lib/commands/new/buildsystems/cli/transpilers/babel.js#L16)

**Pull request merged**